backend/ht_buses_app/views/routes/route_create.py

```diff
@@ -21,6 +21,8 @@ def route_create(request):
         description = reqBody["route"]["description"]
         is_complete = reqBody["route"]["is_complete"]
         route = Route.routeTables.create(name=name, school_id = school, description = description, is_complete=is_complete)
+        route.color_id = route.id % 51
```

Comment:
might come in handy to move the 51 to a constant variable when you get the chance.

Reply:
lemme go do that :)